### PR TITLE
fix: PageSize to be of type string literal

### PR DESCRIPTION
**Pull request by iamstiil**
Updated PageSize from enum to string literals.

### CODE REVIEW

File: types/pdfmake/index.d.ts [hunk 1]
```diff
@@ -23,56 +23,55 @@ declare module "pdfmake/build/pdfmake" {
...
+        | 'C6'
+        | 'C7'
+        | 'C8'
+        | 'C9'
+        | 'C10'
+        | 'RA1'
+        | 'RA2'
+        | 'RA3'
+        | 'RA4'
+        | 'SRA1
```
Comment (evolkmann) [inline types/pdfmake/index.d.ts]: You missed a `'` here

**Review by evolkmann — CHANGES_REQUESTED** [review]
Hi @iamstiil, thanks for submitting the PR. I am fine with the change, but what was the reason behind it?

File: types/pdfmake/index.d.ts (hunk 1 above)
Comment (iamstiil) [reply types/pdfmake/index.d.ts]: The reason is that the string values are not accessible via `pdfmake.PageSize`. So using `pdfmake.PageSize.A4` resulted in valid linting but compile errors. Using a string instead throws linting errors but does compile.

**Comment by dancrumb** [comment]
I feel like this change is a) potentially a breaking change (unless you're arguing that these would never work) and b) need a test that fails with the old typing and passes with the new typing.

File: types/pdfmake/index.d.ts [hunk 2]
```diff
@@ -23,56 +23,55 @@ declare module "pdfmake/build/pdfmake" {
...
+        | 'SRA2'
+        | 'SRA3'
+        | 'SRA4'
+        | 'EXECUTIVE'
+        | 'FOLIO'
+        | 'LEGAL'
+        | 'LETTER'
+        | 'TABLOID';
 
     enum PageOrientation {
```
Comment (sheetalkamat) [inline types/pdfmake/index.d.ts]: While at it, you probably want to convert PageOrientation to type as well

Comment (sheetalkamat) [inline types/pdfmake/index.d.ts]: While at it, you probably want to convert PageOrientation to type as well

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Pull request closed without merging**